rotkehlchen/tests/unit/globaldb/test_upgrades.py

```diff
@@ -382,6 +394,50 @@ def test_upgrade_v4_v5(globaldb):
         assert 'name' not in columns, 'The name column should not be in the contract_data table'
 
 
+@pytest.mark.parametrize('globaldb_upgrades', [[]])
+@pytest.mark.parametrize('reload_user_assets', [False])
+def test_upgrade_v5_v6(globaldb):
+    """Test the global DB upgrade from v5 to v6"""
+    # Check the state before upgrading
+    assert globaldb.get_setting_value('version', None) == 5
```

Comment:
No need in a version check. It is guaranteed when an upgrade-dedicated database is used